api/src/main/resources/schema.graphqls

```diff
@@ -0,0 +1,129 @@
+scalar UUID
+scalar DateTime
+scalar Json
+
+type Query {
+  datasets: [Dataset]
+  namespace(name: String): Namespace
+}
+
+type Tag {
+  name: String
+  createdAt: DateTime
+  updatedAt: DateTime
+  description: String
+
+  fields: [DatasetField]
+  datasets: [Dataset]
+}
+type Source {
+  type: String
+  name: String
+  createdAt: DateTime
+  updatedAt: DateTime
+  connectionUrl: String
+  description: String
+  datasets: [Dataset]
+}
+type RunStateRecord {
+  transitionedAt: DateTime
+  state: String
+  run: Run
+}
+type RunArgs {
+  createdAt: DateTime
+  args: Json
+  checksum: String
+  run: [Run]
```

Comment:
I think we don't need to expose in the model that RunArgs is a separate entity. This is an implementation details.
It could be just a json field in Run